dask_expr/_collection.py

```diff
@@ -5174,6 +5175,7 @@ def read_fwf(
     )
 
 
+@dataframe_creation_dispatch.register_inplace("pandas")
```

Comment:
For the cudf backend, dask-cudf can register a simple function that calls back into this function with `engine=CudfEngine` (already confirmed that this will work fine). The dask-cudf logic can also check the `filesystem` argument and do something like `read_parquet(..., filesystem="arrow").to_backend("cudf")` (since filesystem="arrow" is currently broken in dask-expr when the backend is `"cudf"`)